test/Dialect/ATen/lenet_fwd.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: npcomp-opt %s -aten-to-std |& FileCheck %s --check-prefix=CHECK-CONVERSION
+// CHECK-CONVERSION-LABEL: @graph
```

Comment:
can we check something more specific?

Reply:
This is mostly an integration check for a larger design.